lib-find-meal/src/main/java/com/kinandcarta/lib/find/meal/view/FindMealFragment.kt

```diff
@@ -28,48 +41,161 @@ class FindMealFragment : Fragment() {
     private lateinit var swipeRefreshLayout: SwipeRefreshLayout
 
     private val sdk = MealsSDK()
-    private val mealsAdapter = MealsAdapter(listOf())
+    private var distanceUnit: DistanceUnit = DistanceUnit.miles
+    private val mealsAdapter = MealsAdapter(listOf(), distanceUnit)
+
+    private lateinit var fusedLocationClient: FusedLocationProviderClient
+    private val permissionResultParser: PermissionResultParser by lazy {
+        PermissionResultParser()
+    }
+    private var lastLocation: Location? = null
+    private var hasSetupLocationUpdates = false
+    private var locationUpdateState = false
+
+    private val locationCallback = object : LocationCallback() {
+        override fun onLocationResult(locationResult: LocationResult?) {
+            lastLocation = locationResult?.lastLocation ?: return
+        }
+    }
 
     companion object {
+        val LOCATION_PERMISSION_REQUEST_CODE = 1
```

Comment:
These can probably be `private` and as a small thing I'd move to either the top or bottom of the class